prompts/sources.go

```diff
@@ -120,14 +126,56 @@ func sourceBaseForm(quickstart *Quickstart) (*QuickstartState, error) {
 		fileLocation = *quickstart.Defaults.SchemaPath
 	}
 
-	if _, err := charm_internal.NewForm(huh.NewForm(
-		getBaseSourcePrompts(quickstart.WorkflowFile, &sourceName, &fileLocation, &authHeader, &authSecret)...),
-		"Let's setup a new source for your workflow.",
-		"A source is a compiled set of OpenAPI specs and overlays that are used as the input for a SDK generation.").
-		ExecuteForm(); err != nil {
+	useSampleSpec := false
+	_, err := charm_internal.NewForm(huh.NewForm(
+		huh.NewGroup(
+			huh.NewConfirm().
+				Title("Do you have an OpenAPI document you can use for generation?").
```

Comment:
nit: I would just make this a bit shorter

Do you have an existing OpenAPI spec?

Something like that

Reply:
Updated but I do wonder about `spec`, `specification`, `schema` or `document` what we should go with